### PR TITLE
blog: Software development from an Android device with Gitpod

### PR DESCRIPTION
**Pull request by axonasif**
## Related Issue(s)

Fixes #

## Release Notes

```release-note
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1858"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1858"></a>

### CODE REVIEW

File: src/routes/blog/software-development-from-an-android-with-gitpod.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+slug: software-development-from-an-android-with-gitpod
+teaserImage: teaser.png
+title: Software development from an Android with Gitpod
+---
+
+<script context="module">
+  export const prerender = true;
+</script>
+
+## Outline/ideas
```
Comment (pawlean) [inline src/routes/blog/software-development-from-an-android-with-gitpod.md:14]: This shouldn't be on the final document.

Comment (axonasif) [reply src/routes/blog/software-development-from-an-android-with-gitpod.md:14]: Yeah, it's there because it's still WIP 😬

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+# Getting started with Gitpod on Android
+
+![my android setup](../../..../../../static/images/blog/software-development-from-my-android-phone-at-datacenter-speeds/20220412_004915681.jpg)
```
Comment (Nancy-Chauhan) [inline src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:33]: Hey @axonasif Let's please put a caption here

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+Getting started with Gitpod on Android is quite easy. If you are interested in setting up Gitpod on Android, you can check out the detailed guide: [draft](https://docs.google.com/document/d/18Y4fRCcO0pCGXrMbxNj-xqkcymn0laTvJep5Zhtc5UA/edit#)
+
+insert youtube video link: [draft on slack](https://gitpod.slack.com/archives/C01LDA27WJC/p1654875261466739?thread_ts=1654626018.462279&cid=C01LDA27WJC)
```
Comment (Nancy-Chauhan) [inline src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:37]: Can you simply insert a youtube video! ( example:https://www.gitpod.io/blog/gitpod-jetbrains#getting-started-in-5-minutes)

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+# How I got an Internship at Gitpod
+
+When I came across Gitpod, I applied for the **[Professional Open Source](https://www.gitpod.io/docs/professional-open-source)** plan after running out of free hours. Surprisingly, I was found eligible for it! 🥳
+
+As a kid, it inspired me! All my tiny little efforts weren't worthless afterall. Since then, I have strongly wanted to give something back to the Gitpod Community. I started hanging out with the Gitpod community and helping others whenever possible. It made me feel happy. I consider this as a method of learning things as you get presented with new challenges everyday.
+
+Around that time Gitpod also launched the [Community Heroes program](https://www.gitpod.io/community/heroes) - an initiative that recognises the most active community members that go above and beyond to contribute to community 🍊🦸. I got recognised for my consistent efforts and was a member of the first cohort 😎
+
+[![community heros](../../../static/images/blog/software-development-from-my-android-phone-at-datacenter-speeds/community_heros_tweet.png)](https://twitter.com/gitpod/status/1466452229309837313?s=20&t=3RxZdUpS8QSMn601harwNQ)
```
Comment (Nancy-Chauhan) [inline src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:62]: @axonasif can we embed the tweet. If not, please adjust the size of the image. It should align with the text and size should be smaller

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+Later, I had applied for an internship at Gitpod. It was kind of an unexpected event for me: I got the internship, and officially, I became a Gitpodder 🍊
+
+[![Mission passed: RESPECT 😝](../../../static/images/blog/software-development-from-my-android-phone-at-datacenter-speeds/internship_tweet.png)](https://twitter.com/axonasif/status/1478418230460907529?s=20&t=3RxZdUpS8QSMn601harwNQ)
```
Comment (Nancy-Chauhan) [inline src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:66]: Same here! Need changes

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+# Unlocking the power of Gitpod
+
+Can you believe it? All the contributions and projects have been made from my phone, using the power of Gitpod! Gitpod is powerful, and I believe with Gitpod, anyone can learn to code and develop software without any hurdles.
+
+Drop by our community Discord server if you find this interesting and want to develop software on your phone. We have the[ #mobile-and-tablets](https://discord.com/channels/816244985187008514/890901203624534026) channel where fellow mobile-dev enthusiasts hang out! Feel free to share your tips and tricks on software engineering in a more unconventional but accessible way.
```
Comment (Nancy-Chauhan) [inline src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:72]: `We have the[ #mobile-and-tablets]`

Insert space b/w the and [ #mobile-and-tablets]

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+- Step 2: Install the [Gitpod extension](https://chrome.google.com/webstore/detail/gitpod-online-ide/dodmmooeoklaejobgleioelladacbeki) in Kiwi browser 🍊
+- Step 3: Open a Git repository in Gitpod ⚡️
+- Step 4: Code, Develop, Ship it 🚀
+
+# Things I built/maintained with Gitpod on my Android phone
+
+Here are some fun things I built on mobile:
+
+- [Optimus Discord Bot](https://github.com/supremegamers/optimus) - This hackathon project is now powering our Gitpod Community Discord server 🎉 I started developing the first version of it for [https://aopc.dev/](https://aopc.dev/) discord server before I knew about Gitpod. The low-end Notebook, which I initially used eventually died and was unsuitable for this task. Later, I tried working on my phone in a chrooted Linux environment, but the compile-time for this project made it impossible for me to develop and made my phone super hot. That's when I discovered Gitpod and started using it. It made the development super easy!
+- [https://github.com/axonasif/gearlock](https://github.com/axonasif/gearlock) - Maintaining a bootable custom-recovery and package manager for Android-x86.
```
Comment (pawlean) [inline src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:50]: **Small nit:** Can you add the title of each GitHub Project here? Similar to what you've done for Optimus Discord Bot. Let's keep that consistent!

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+- [https://github.com/supremegamers/darkmatter](https://github.com/supremegamers/darkmatter) - Maintaining a customized Android-x86 distro.
+- [https://github.com/axonasif/multitux-gp](https://github.com/axonasif/multitux-gp) - Attempt to parse Gitpod `workspace-images` Dockerfiles and generate them for different distros. It's still a work in progress!
+
+# How I got an Internship at Gitpod
+
+When I came across Gitpod, I applied for the **[Professional Open Source](https://www.gitpod.io/docs/professional-open-source)** plan after running out of free hours. Surprisingly, I was found eligible for it! 🥳
+
+As a kid, it inspired me! All my tiny little efforts weren't worthless afterall. Since then, I have strongly wanted to give something back to the Gitpod Community. I started hanging out with the Gitpod community and helping others whenever possible. It made me feel happy. I consider this as a method of learning things as you get presented with new challenges everyday.
```
Comment (pawlean) [inline src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:59]: I'd rephrase this to "as someone younger, it inspired me!" 🙌

**Review by pawlean — COMMENTED** [review]
LGTM apart from the small comments 🙌 

Let's also make sure when we're shipping this is updated on [our launch calendar.](https://www.notion.so/gitpod/935881775fc547daacb359f70939e170?v=aa9b43c05af04058a2584e2ef2bf0e37)

Great job, looking forward to seeing this live!

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md (hunk 8 above)
Comment (axonasif) [reply src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:59]: To me that(`younger`) sounds kinda unusual 😅
In the case of rephrasing it, could we do
```
That really inspired me! All my tiny little projects weren't worthless afterall
```

File: src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md (hunk 7 above)
Comment (axonasif) [reply src/routes/blog/software-development-from-my-android-phone-at-datacenter-speeds.md:50]: Done

**Review by pawlean — APPROVED** [review]
LGTM

**Review by Nancy-Chauhan — APPROVED** [review]
